File: Day 1/Day 1/challenge1.py
#challenge 1

t = int(input())

for i in range(t):
    	s = input()
    	o = 0
    	b = 0
    	for j in s:
        	if j == 'o':
            		o += 1
        	elif j == 'b':
            		b += 1
    	if o >= b:
        	print(b)
    	else:
        	print(o)

File: Day 1/Day 1/challenge1.c
//Challenge 1(Day 1)

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
	int t;
    	char a[30];
    	scanf("%d",&t);
    	for(int i=0;i<t;i++)
    		{ 
		int counto=0,countb=0,l;
     		scanf("%s",a);
      		l= strlen(a);
      		for(int j=0;j<l;j++)
     		 { 
		if(a[j]=='o')
          		counto++;
       		else if(a[j]=='b')
           		countb++;
                
     				 }
     	if(countb==counto)
         	printf("%d \n",countb);
    	else if (countb<counto)
         	printf("%d \n",countb);
     	else  if(counto<countb)
           	printf("%d \n",counto);
                    
    				}     
    return 0;
}